src/OracleService/Protocols/OracleHttpsProtocol.cs

```diff
@@ -12,6 +14,13 @@ class OracleHttpsProtocol : IOracleProtocol
     {
         private readonly HttpClient client = new HttpClient();
 
+        public OracleHttpsProtocol()
+        {
+            CustomAttributeData attribute = Assembly.GetExecutingAssembly().CustomAttributes.First(p => p.AttributeType == typeof(AssemblyInformationalVersionAttribute));
+            string version = (string)attribute.ConstructorArguments[0].Value;
+            client.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue("Neo Oracle Service", version));
```

Comment:
```suggestion
            client.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue("NeoOracleService", version));
```
Should never add any blank in User-Agent, otherwise will be format error.
![image](https://user-images.githubusercontent.com/38493437/114121056-49238580-9920-11eb-9479-731dee28fb4e.png)